blog/2023-ecosystem-eoy-recap.md

```diff
@@ -0,0 +1,122 @@
+    baseUrl: `https://my-manifest.url/${process.platform}/${process.arch}`,
+  },
+});
+```
+
+:::info Further reading
+
+📦 Want to learn more? For a detailed guide, see [Forge’s auto update documentation](https://www.electronforge.io/advanced/auto-update).
+
+:::
+
+## The `@electron/` extended universe
+
+When Electron first started, the community published many packages to enhance the experience of developing, packaging, and distributing Electron apps. Over time, many of these packages were incorporated into Electron’s GitHub organization, with the core team taking on the maintenance burden.
+
+In 2022, we began unifying all these first-party tools under the `@electron/` namespace on npm. This change means that packages that used to be `electron-foo` are now `@electron/foo` on npm, and repositories that used to be named `electron/electron-foo` are now `electron/foo` on GitHub. These changes help clearly delineate first-party projects from userland projects. This includes many commonly used packages, such as:
+
+- `@electron/asar`
+- `@electron/fuses`
+- `@electron/get`
+- `@electron/notarize`
+- `@electron/osx-sign`
+- `@electron/packager`
+- `@electron/rebuild`
+- `@electron/remote`
+- `@electron/symbolicate-mac`
+- `@electron/universal`
+
+Going forward, all first-party packages we release will also be in the `@electron/` namespace. There are two exceptions to this rule:
+
+- Electron core will continue to be published under the `electron` package.
+- Electron Forge will continue to publish all of its monorepo packages under the `@electron-forge/` namespace.
+
+:::info Star seeking
+
+⭐ During this process, we also accidentally took the electron/packager repository private, which has the unfortunate side effect of erasing our GitHub star count (over 9000 before the erasure). If you are an active user of Packager, we’d appreciate a ⭐ **Star** ⭐!
+
+:::
+
+## Introducing `@electron/windows-sign`
+
+Starting on 2023-06-01, industry standards began requiring keys for Windows code signing certificates to be stored on FIPS-compliant hardware.
+
+In practice, this meant that code signing became a lot harder for apps that build and sign in CI environments, since many Electron tools take in a certificate file and password as config parameters and attempt to sign from there using hardcoded logic.
+
+This situation has been a common pain point for Electron developers, which is why we have been working on a better solution that isolates Windows code signing into its own standalone step, similar to what `@electron/osx-sign` does on macOS.
+
+In the future, we plan on fully integrating this package into the Electron Forge toolchain, but it currently lives on its own. The package is currently available for installation at `npm install --save-dev @electron/windows-sign` and can used programmatically or via CLI.
+
+Please try it out and give us your feedback in the [repo’s issue tracker](https://github.com/electron/windows-sign/issues)!
+
+## What's Next?
```

Comment:
```suggestion
## What's next?
```

Very nit, but we generally use sentence case for headings like this.

Reply:
Good catch, fixed